lang/en/tool_crawler.php

```diff
@@ -82,15 +82,15 @@
 $string['eventrobotcleanupcompleted'] = 'Linkchecker robot cleanup completed';
 $string['eventrobotcleanupstarted'] = 'Linkchecker robot cleanup started';
 $string['excludeexturl'] = 'Exclude external URLs';
-$string['excludeexturldesc'] = 'One URL regex per line. Each is matched against the full URL.';
+$string['excludeexturldesc'] = 'One URL per line. Each is matched against the full URL.';
```

Comment:
Can you please also mention the wildcard char in both help strings eg

$string['profilingincluded_help'] = 'List of (comma or newline separated, absolute skipping wwwroot, callable) URLs that will be automatically profiled. Examples: /index.php, /course/view.php. Also accepts the * wildchar at any position. Examples: /mod/forum/*, /mod/*/view.php.';

Reply:
I have made the requested changes.